File: 00_Learn_The_Basics/Things_To_Know_In_A_Language/If-Else/evenodd.cpp
#include<iostream>
using namespace std;

int main()
{
    int n;
    cout<<"Enter the number to be checked whether it is even or odd "<<endl;
    cin>>n;

    if(n%2==0)
    cout<<"The number "<< n << " is even "<< endl;
    else
    cout<<"The number "<< n << " is not even "<< endl;
}

File: 00_Learn_The_Basics/Things_To_Know_In_A_Language/If-Else/EvenOdd.java
//Finding whether the number entered by the user is even or odd

import java.util.*;

public class EvenOdd
{
    public static void main(String[] args) 
    {
        Scanner in= new Scanner(System.in);
        
        System.out.println("Enter the number to be checked whether it is even or odd");
        
        int n=in.nextInt();

        if(n%2==0)
            System.out.println("The number " + n +" is even");

        else
            System.out.println("The number " + n +" is odd");

    }
}
